lib/osf-components/addon/components/registries/schema-block-group-renderer/template.hbs

```diff
@@ -8,6 +8,7 @@
                 @renderStrategy={{@renderStrategy}}
                 @isRequired={{this.isRequired}}
                 @isFieldsetGroup={{true}}
+                @draftManager={{@draftManager}}
```

Comment:
```suggestion
```
now that `draftManager` is held by the `renderStrategy`, don't need it here (unless i'm missing something)